crates/shim-protos/Cargo.toml

```diff
@@ -39,4 +43,16 @@ path = "examples/ttrpc-client.rs"
 
 [[example]]
 name = "shim-proto-connect"
-path = "examples/connect.rs"
\ No newline at end of file
+path = "examples/connect.rs"
+
+[[example]]
+name = "shim-proto-server-async"
+path = "examples/ttrpc-server-async.rs"
```

Comment:
Please add `required-features = ["async"]` to enable this example only when feature async is enabled.
https://doc.rust-lang.org/cargo/reference/cargo-targets.html#the-required-features-field

Reply:
good point, already resolved, thanks